frontend/src/components/ModelBox/ModelConfiguration.tsx

```diff
@@ -1,17 +1,24 @@
 import { useEffect, useState } from 'react';
 
-import { CustomChatModelConfiguration, MODEL_CONFIG } from '@src/models/chat';
+import {
+	ChatModel,
+	CustomChatModelConfiguration,
+	MODEL_CONFIG,
+} from '@src/models/chat';
 import { chatService } from '@src/service';
 
 import ModelConfigurationSlider from './ModelConfigurationSlider';
 
 import './ModelConfiguration.css';
 
 function ModelConfiguration({
+	chatModel,
 	addInfoMessage,
 }: {
+	chatModel?: ChatModel;
 	addInfoMessage: (message: string) => void;
 }) {
+	// can we move the default configurations to a different file?
```

Comment:
Either that or just outside of the component itself. Doesn't _need_ to be in a different file.

Reply:
moved out of the component